lancero_source.go

```diff
@@ -393,8 +405,18 @@ func (ls *LanceroSource) distributeData(timestamp time.Time, wait time.Duration)
 		// mask out frame and extern trigger bits from FB channels
 		if channum%2 == 1 {
 			const mask = ^RawType(0x3)
-			for j := 0; j < len(data); j++ {
-				data[j] &= mask
+			mixFraction := ls.MixFraction[channum]
```

Comment:
One point about mixing that you didn't know: the `ndfb_server` doesn't mix the current error with the current FB value. It mixes the _previous_ error into the current FB. (Or do I have that backwards??) I think the argument for doing this is both that it's more correct in some sense and that it is needed to match what we have been doing in past DAQ code.

Unfortunately, this involves special handling for the first sample, and saving one last error sample for each channel from call to call. Ugh!

Reply:
Yes i agree it is more annoying, and you're probably right that it is more correct. I'll double check with Carl, and think about how to handle it.

Reply:
Also I just went for it with the default plus override, I really hoped it would work, and though it might work, and was pleasantly surprised.